internal/provider/serverless_cluster.go

```diff
@@ -175,3 +240,15 @@ func resourceServerlessClusterDelete(ctx context.Context, d *schema.ResourceData
 
 	return nil
 }
+
+func getFreeTierHoursAvailable(c *scylla.Client, ctx context.Context) (int, error) {
+	info, err := c.GetAccountInfo(ctx)
+	if err != nil {
+		return 0, fmt.Errorf("error getting account into: %s", err)
+	}
+	freeTierDeal := info.GetFreeTierDeal()
+	if !freeTierDeal.IsEligible || freeTierDeal.Total <= freeTierDeal.Used {
+		return 0, nil
+	}
+	return int(freeTierDeal.Total - freeTierDeal.Used), nil
```

Comment:
side question, but do we really expect api users to perform such computations? 🤔

Reply:
Cons:
- Otherwise it is going to fail with error that user can't interpret.
- UI do the same here
- After all it is not for production cluster, we can afford loosing a bit of consistency for sake of user experience